### PR TITLE
Feat/pr-templates

### PR DESCRIPTION
**Pull request by andresin87**
Adds a template by url
`https://github.com/SUI-Components/sui-components/compare/master...[branch pushed]&template=feature.md`

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/feature.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#### [LEVEL]/[COMPONENT]
+| TASK | <!--- [jira TASK ID](jiraURL) -->             |
+| ---- | --------------------------------------------- |
+| DEMO | <!--- [[LEVEL]/[COMPONENT]](zeit now URL) --> |
```
Comment (jcger) [inline .github/PULL_REQUEST_TEMPLATE/feature.md]: not sure if we really need this as now is going to add a message with the link as soon as it's deployed

File: .github/PULL_REQUEST_TEMPLATE/feature.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+##### Description, Motivation and Context
+<!--- Describe your changes in detail -->
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
```
Comment (jcger) [inline .github/PULL_REQUEST_TEMPLATE/feature.md:16]: what about asking for how to reproduce in case it's a bugfix?

Comment (andresin87) [reply .github/PULL_REQUEST_TEMPLATE/feature.md:16]: `If it fixes an open issue, please link to the issue here.` Is removed, sorry

File: .github/PULL_REQUEST_TEMPLATE/feature.md (hunk 1 above)
Comment (andresin87) [reply .github/PULL_REQUEST_TEMPLATE/feature.md]: I supose that if we specify in the title the demo, maybe this direct link is not needed as you suggest

File: .github/PULL_REQUEST_TEMPLATE/feature.md (hunk 2 above)
Comment (andresin87) [reply .github/PULL_REQUEST_TEMPLATE/feature.md:16]: Ok, I'll add it too

**Review by jcger — APPROVED** [review]

**Review by midudev — APPROVED** [review]

**Review by nucliweb — APPROVED** [review]
What about to promote to add a screenshot or gif animation?

File: .github/PULL_REQUEST_TEMPLATE/feature.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+##### Description, Motivation and Context
+<!--- Describe your changes in detail -->
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it is solving an issue... How can it be reproduced in order to compare between both behaviors? -->
+
+##### Screenshots (if appropriate):
```
Comment (turolopezsanabria) [inline .github/PULL_REQUEST_TEMPLATE/feature.md]: I suggest removing the phrase "if appropriate" as it reduces the perceived value of adding examples of your changes.

As a reinforcement I would add a comment, something like 
"Adding images or gif animations of your changes improves the understanding  of your changes"

**Review by turolopezsanabria — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/feature.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+##### Description, Motivation and Context
+<!--- Describe your changes in detail -->
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it is solving an issue... How can it be reproduced in order to compare between both behaviors? -->
+
+##### Screenshots - Animations:
```
Comment (nucliweb) [inline .github/PULL_REQUEST_TEMPLATE/feature.md]:
```suggestion
##### Screenshots - Animations
```

**Pull request merged**